BUILD.plz

```diff
@@ -1,7 +1,12 @@
+subinclude("//build_defs:version")
```

Comment:
Keeping the `VERSION` file in the root of the repo  and defining a `filegroup` for it causes a circular dependency because `//build_defs:version` requires that `filegroup`, and the `go_binary` requires `//build_defs:version`.

I see two possibilities:
   - move the wollemi binary up a directory.
      * This would allow the VERSION file to remain in the root of the repo, which is where most people would expect it. 
      * not likely to be forgotten about.
   - avoid all the genrule and build_def steps and hard code the version in the go_binary definitions attribute.
      * no major changes needed for this
      * could be easy to forget about this and forget to increment the version on a new release
      * less clear where to increment the version for new contributors